resolwe/flow/executors/__main__.py

```diff
@@ -0,0 +1,18 @@
+"""Main standalone execution stub, used when the executor is actually run."""
+
+import sys
+from importlib import import_module
+
+from .run import settings
+
+
+def run_executor():
+    """Start the actual execution; instantiate the executor and run."""
+    module_name = '.'.join(sys.argv[1].split('.')[:-1])
+    class_name = sys.argv[1].split('.')[-1]
+    module = import_module(module_name, __package__)
+    executor = getattr(module, class_name)()
+    executor.run(settings['Data_get']['id'], open('task.script', 'rt').read(), settings['verbosity'])
```

Comment:
Can you move the `'task.script'` string into some constant and use that instead?

Reply:
Also what is the rationale behind having a setting named `Data_get` (the "get" part)? Are there any other things inside the `Data` namespace (planned)?

Reply:
As I've commented later and we discussed in person, separate the monolithic `settings.json` into `settings.json`, `data.json`, `data_meta`.json and `process.json`, which will make the `Data_get` part obsolete. You can then just load `data.json` into the `data` variable.